fmn/consumer/backends/mail.py

```diff
@@ -151,9 +151,11 @@ def _format_line(msg):
         n = len(queued_messages)
         subject = u"Fedora Notifications Digest (%i updates)" % n
         summary = u"Digest summary:\n"
+        i = 0
         for msg in queued_messages:
+            i = i + 1
             line = fedmsg.meta.msg2subtitle(msg, **self.config) or u''
-            summary += str(i+1) + ".\t" + line + "\n"
+            summary += str(i) + ".\t" + line + "\n"
```

Comment:
Instead of the above, how about:

```
for i, msg in queued_messages:
    …
    summary += "{}.\t{}\n".format(str(i + 1), line)
```

Or you can use the %s format, since I know you like that ☺ ```enumerate``` is a nicer way to generate indices in for loops though, and you can drop two lines of code!